src/data/markdown/docs/01 guides/04 Results visualization/00 End-of-test Summary.md

```diff
@@ -0,0 +1,199 @@
+---
+title: 'End-of-test Summary'
+excerpt: 'By default, k6 prints a summary report containing the aggregated results at the end of the test. Since k6 v0.30.0, the new `handleSummary() callback allows the report to be completely customized, including the generation and saving of JSON, HTML, XML (e.g. JUnit), etc. reports to files.'
```

Comment:
```suggestion
excerpt: 'By default, k6 prints a summary report containing the aggregated results at the end of the test. Since k6 v0.30.0, the new `handleSummary()` callback allows the report to be completely customized, including the generation and saving of JSON, HTML, XML (e.g. JUnit), etc. reports to files.'
```